civihr_employee_portal/templates/civihr-employee-portal-custom-reports.tpl.php

```diff
@@ -105,6 +105,9 @@ class="btn btn-lg btn-secondary-outline text-uppercase">
             </div>
         </footer>
     </section>
+    <div data-graph-section="summary">
```

Comment:
I don't know exactly how everything is build in reports, but is it necessary to insert all the HTML via Javascript?